manifests/repo.pp

```diff
@@ -0,0 +1,26 @@
+class squid::repo
+{
+  if $::squid::manage_repo {
+    case $::osfamily {
+      'RedHat': {
+        case $::operatingsystemmajrelease {
+          '6','7': {
+            yumrepo { 'squid':
+              name     => 'Squid',
+              descr    => 'Squid repo for CentOS Linux - $basearch',
+              baseurl  => 'http://www1.ngtech.co.il/repo/centos/$releasever/$basearch/',
```

Comment:
@alexjfisher You should use CentOS repo for CentOS and RHEL for RHEL.
Despite to the fact that these two are compatible I found myself couple times wondering about couple little things.
The issue is that since I have not updated my RHEL build node I have not published another version since.
However since it's mostly dynamic libraries that are different I will try to build and if everything will look fine I will updated the RHEL repository.